environment

```diff
@@ -104,33 +114,35 @@ DSS_AWS_FLASHFLOOD_PREFIX_READ_PROD="aws"
 DSS_AWS_FLASHFLOOD_PREFIX_WRITE_PROD="aws"  # This can be a comma separated list
 DSS_GCP_FLASHFLOOD_PREFIX_READ_PROD="gcp"
 DSS_GCP_FLASHFLOOD_PREFIX_WRITE_PROD="gcp"  # This can be a comma separated list
-DSS_INFRA_TAG_PROJECT=dcp
-DSS_INFRA_TAG_SERVICE=dss
-DSS_INFRA_TAG_OWNER=dss-team@data.humancellatlas.org
-DSS_NOTIFICATION_SENDER="DSS Notifier <dss.humancellatlas@gmail.com>"
-ADMIN_USER_EMAILS=""
-DCP_DOMAIN=${DSS_DEPLOYMENT_STAGE}.data.humancellatlas.org
-API_DOMAIN_NAME="dss.${DCP_DOMAIN}"
+
+DSS_INFRA_TAG_PROJECT="cedar"
+DSS_INFRA_TAG_SERVICE="dss"
+
+DSS_INFRA_TAG_OWNER="chmreid@ucsc.edu"
+DSS_NOTIFICATION_SENDER="chmreid@ucsc.edu"
+API_DOMAIN_NAME="dss.${STAGE}.chmreid.ucsc-cgp-redwood.org"
+
 TOKENINFO_FUNC=dss.util.security.verify_jwt
 # TODO remove https://dev.data.humancellatlas.org/ from OIDC_AUDIENCE once seperate deployments are made
-OIDC_AUDIENCE=https://dev.data.humancellatlas.org/,https://${DCP_DOMAIN}/
+OIDC_AUDIENCE=https://dev.data.humancellatlas.org/,https://dev.data.humancellatlas.org/
```

Comment:
`https://auth.ucsc.ucsc-cgp-redwood.org`

Reply:
fixed in dad3a4a